File: Modul - 6/Soal - 3/PRAK603-2410817220009-NABILLAPUTRINUGRAHA.c
#include <stdio.h>
int main() {
    int n1, n2;

    //buat nginput n1 sama n2
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama \n");
    }
    else {
        int baris1[n1], baris2[n2], result[n1];
    
    //buat nginput angka baris pertama
    for(int i = 0; i < n1; i++) {
        scanf("%d", &baris1[i]);
    }

    //buat nginput baris kedua
    for(int i = 0; i < n2; i++) {
        scanf("%d", &baris2[i]);
    }

    //perkalian
    for(int i = 0; i < n1; i++) {
        result[i] = baris1[i] * baris2[i];
    }

    //buat nyetak hasil
    printf("\n");
    for(int i = 0; i < n1; i++) {
        printf("%d ", result[i]);
    }
    printf("\n");
    }
    return 0;

}

File: Modul - 6/Soal - 3/PRAK603-2410817220009-NABILLAPUTRINUGRAHA.py
#fungsi buat perkalian dua baris
def kali_baris(baris1, baris2) :
    return[baris1[i] * baris2[i] for i in range (len(baris1))]

#buat nginput bilangan n1 dan n2
n1, n2 = map(int, input(" ").split())

if n1 != n2 :
    print("Jumlah tidak sama")
else :
    #buat nginput baris pertama
    baris1 = list(map(int, input().split()))

    #buat nginput baris kedua
    baris2 = list(map(int, input().split()))

    #perkalian
    result = kali_baris(baris1, baris2)

    print()
    #nyetak hasil
    print(' '.join(map(str,result)))
